ui/app/adapters/kv/data.js

```diff
@@ -38,9 +38,8 @@ export default class KvDataAdapter extends ApplicationAdapter {
     });
   }
 
-  fetchSubkeys(query) {
-    const { backend, path, version, depth } = query;
-    const url = this._url(kvSubkeysPath(backend, path, depth, version));
+  fetchSubkeys(backend, path, query) {
```

Comment:
while here I also changed how these args are passed, I think this pattern better aligns with how we use adapter methods (by passing the query as an object, not all of the args)